content/calls/workshop_call.md

```diff
@@ -26,22 +24,21 @@ Workshop proposals should be no more than 3 pages long and should include the fo
-- Compiling a camera-ready copy for the workshop proceedings in close interaction with the workshop program chairs. The workshop proceedings will be made available on the ICAPS 2023 website.
+-  Producing a Call for Papers (CFP) and contributing materials for setting up a web page for the workshop by 5 February . The web page and the CFP will be linked to the ICAPS 2024 website. Details on this process will be made available after notification.
+-  Additional publicity, such as distributing the CFP to relevant newsgroups and electronic mailing lists (through liaison with the ICAPS 2024 Publicity Chair) and especially to potential audiences outside the ICAPS community.
+-  Organizing the review process for submissions and issuing invitations to participate in the workshop.
+-  Formulating a detailed technical program for the workshop and making it available to be uploaded on the ICAPS 2024 website well in advance of the conference. A list of accepted papers for the workshop should be ready to be placed on the workshop website by 29 April 2024.
+-  Compiling a camera-ready copy for the workshop proceedings in close interaction with the workshop program chairs. The workshop proceedings will be made available on the ICAPS 2024 website.
+
 
 ## Submission Procedure
-Workshop proposals should be submitted by email, in plain text or PDF, to icaps2023-workshops@googlegroups.com with the subject “Workshop Proposal for ICAPS23” no later than 6 January 2023. Workshop organizers will be notified by email of the acceptance or rejection of their proposal by 27 January 2023. All PDFs must be less than 25 MB in size.
+Workshop proposals should be submitted by email, in plain text or PDF, to icaps2024-workshops@googlegroups.com with the subject “Workshop Proposal for ICAPS2024” no later than 12 January 2024. Workshop organizers will be notified by email of the acceptance or rejection of their proposal by 29 January 2024. All PDFs must be less than 25 MB in size.
 
 ## Summary of Important Dates:
-- ***Deadline for submission of workshop proposals***: ~~06 January 2023~~ ***13 January 2023***.
-- Notification of workshop acceptance:  27 January 2023.
-- Workshop Call for Papers and materials for the website: 03 February 2023.
-- List of accepted papers on the workshop website: 28 April 2023.
-- Workshop/Tutorial dates: 09-10 July 2023.
-
-Please direct all questions to the Workshop Program Chairs at <icaps2023-workshops@googlegroups.com>
-
+- Deadline for submission of workshop proposals: 12 January 2024.
+- Notification of workshop acceptance: 29 January 2024.
+- Workshop Call for Papers and materials for the website: 5 February 2024
+- List of accepted papers on the workshop website: 29 April 2024.
+- Workshop/Tutorial dates: 01-02 June 2024.
 
+Please direct all questions to the Workshop Program Chairs at icaps2024-workshops@googlegroups.com
```

Comment:
We've set up the following alias for you already -- might be good to keep them uniform, unless you want the groups functionality?

workshops@icaps2024.com
- Biplav
- Shirin

Reply:
I am inclined to keep the one we created.